content/intro-to-storybook/svelte/en/test.md

```diff
@@ -0,0 +1,154 @@
+
+## Visual regression testing for Storybook
+
+Visual regression tests are designed to catch changes in appearance. They work by capturing screenshots of every story and comparing them commit-to-commit to surface changes. This is perfect for verifying graphical elements like layout, color, size, and contrast.
+
+<video autoPlay muted playsInline loop style="width:480px; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/visual-regression-testing.mp4"
+    type="video/mp4"
+  />
+</video>
+
+Storybook is a fantastic tool for visual regression testing because every story is essentially a test specification. Each time we write or update a story we get a spec for free!
+
+There are a number of tools for visual regression testing. For professional teams we recommend [**Chromatic**](https://www.chromaticqa.com/), an addon made by Storybook maintainers that runs tests in the cloud.
+
+## Setup visual regression testing
+
+Chromatic is a hassle-free Storybook addon for visual regression testing and review in the cloud. Since it’s a paid service (with a free trial), it may not be for everyone. However, Chromatic is an instructive example of a production visual testing workflow that we'll try out for free. Let’s have a look.
+
+### Initialize local repository
+
+At this stage there's still no repository setup just yet. Let's change that and set it up:
+
+```bash
+$ git init 
+```
+
+Next add files to the first commit.
+
+```bash
+$ git add -A
+```
+
+Now commit the files.
+
+```bash
+$ git commit -m "taskbox UI"
+```
+
+### Get Chromatic
+
+Add the package as a dependency.
+
+```bash
+npm install -D storybook-chromatic
+```
+
+Import Chromatic in your `.storybook/config.js` file.
+
+```javascript
+// .storybook/config.js
+
+import { configure } from '@storybook/react';
+import requireContext from 'require-context.macro';
+import 'storybook-chromatic';
+
+import '../src/index.css';
+
+const req = requireContext('../src/components', true, /\.stories\.js$/);
+
+function loadStories() {
+  req.keys().forEach(filename => req(filename));
+}
+
+configure(loadStories, module);
+```
+
+Then [login to Chromatic](https://www.chromaticqa.com/start) with your GitHub account (Chromatic only asks for lightweight permissions). Create a project with name "taskbox" and copy your unique `app-code`.
+
+<video autoPlay muted playsInline loop style="width:520px; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/chromatic-setup-learnstorybook.mp4"
+    type="video/mp4"
+  />
+</video>
+
+Run the test command in the command line to setup visual regression tests for Storybook. Don't forget to add your unique app code in place of `<app-code>`.
```

Comment:
We need to add the `-s` option to the `build-storybook` script here (or maybe we should do it at the start when we do it to the `storybook` script?

Reply:
My example code is using the following script for building storybook:
```json
{
 "scripts":{
   .....
"build-storybook": "build-storybook -c .storybook -s public -o storybook-static",
}
}
Is that ok with you?

Reply:
Sure, although I think the `-c` is not necessary (that's the default) and the `-o` doesn't necessarily matter for this tutorial. So it might be better to keep things simple?

Reply:
I've added some changes to the document to address this. See if the wording is ok with you and let me know.